src/classes/GE_PaymentServices.cls

```diff
@@ -130,14 +159,21 @@ public with sharing class GE_PaymentServices {
             generator.writeStringField('iss', 'ApiKeyManager'); // Issuer
             generator.writeNumberField('exp', getExpirationDate(1)); // Expires at
             generator.writeStringField('sub', UserInfo.getUserName()); // Subject
-            generator.writeStringField('sfdo.id', 'bees'); // Org ID
+            generator.writeStringField('sfdo.id', 'bees');
```

Comment:
This should be the elevate onboarding "productId" parameter I believe - we are currently receiving this from int1

Reply:
Ok, I'll update the code and the script to insert this value into test config.

Reply:
fixed!